Description: Add keywords to the tellico.desktop file
 Add keywords to the tellico.desktop file

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

--- tellico-2.3.8+dfsg.1.orig/tellico.desktop
+++ tellico-2.3.8+dfsg.1/tellico.desktop
@@ -135,4 +135,5 @@ GenericName[x-test]=xxCollection Manager
 GenericName[zh_CN]=收藏管理器
 GenericName[zh_TW]=收藏管理員
 MimeType=application/x-tellico;
+Keywords=collection;books;wine;coins;
 Categories=KDE;Qt;Office;Database;
